app/controllers/users_controller.rb

```diff
@@ -0,0 +1,100 @@
+class UsersController < ApplicationController
+  include ApplicationHelper
+
+  before_action :set_user, only: [:show, :edit, :update, :destroy]
+  before_action :logged_in_user, only: %i[check_invites]
+  SQL = 'INNER JOIN events ON events.id = rsvps.attended_event_id'
+  # GET /users
+  # GET /users.json
+  def index
+    @users = User.all
+  end
+
+  # GET /users/1
+  # GET /users/1.json
+  def show
+    @user = User.find(params[:id])
```

Comment:
Why don't you use the `current_user` you already created?

Reply:
[OPTIONAL]
Yes, please consider using `current_user`. This is an unnecessary query.